### ISSUE TITLE
[Reader] The reader_accessed event is tracked when you return to the posts list from the comments screen

### ISSUE THREAD

**Issue by daniloercoli** [wordpress-mobile/WordPress-Android#20441]
The `reader_accessed` event is tracked when you return to the posts list from the comments screen.
Steps to reproduce:
   * Open the Reader.
   * Tap on an article with at least one comment to open the details view.
   * Tap on "View all comments."
   * On the comments screen, tap back to return to the posts list screen.
   * The reader_accessed event is tracked again.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by RenanLukas** [comment]
Blocked by discussion p1710259059172569-slack-C01CW1VMLAF

**Comment by RenanLukas** [comment]
Won't fix p1710259059172569-slack-C01CW1VMLAF
